databases/functions/fcmToken/find-token-by-user-id.js

```diff
@@ -10,6 +10,7 @@ module.exports = async (fcmTokenModel, userId) => {
     where: {
       user_id: userId
     },
+    order: [['updated_at', 'DESC']],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `order` clause is a good practice to ensure that the most recent FCM token is used. However, it's important to handle cases where there are no tokens available for a user. Make sure to add error handling after this query to check if a token was actually returned.
```diff
     where: {
       user_id: userId
     },
     order: [['updated_at', 'DESC']],
     raw: true
   });

+  if (!token) {
+    throw new Error('No FCM token found for user');
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->